samples/README_Windows.md

```diff
@@ -47,7 +47,7 @@ set OpenEnclave_DIR=C:\openenclave\lib\openenclave\cmake
 ```cmd
 mkdir build
 cd build
-cmake .. -G Ninja -DNUGET_PACKAGE_PATH=C:\your\path\to\intel_and_dcap_nuget_packages
```

Comment:
There is one more `your\path\to` in `building_oe_sdk.md`

Reply:
Thanks @jxyang. Fixed.